screenpy/actions/eventually.py

```diff
@@ -3,7 +3,7 @@
 """
 
 import time
-from typing import Optional, Set, Tuple
+from typing import Dict, Optional
```

Comment:
You don't actually need to import `Dict` for typehinting, you can just use `dict` directl—

oh wait, backwards compatible to 3.8. 🤦